README.md

```diff
@@ -1,3 +1,27 @@
 # sublime_lib
 
-Utility library for frequently used functionality in Sublime Text library and convenience functions or classes
+An unofficial "standard library" for Sublime Text. Provides a variety of useful convenience features for other packages.
```

Comment:
I don't think we need to declare it as "unofficial" when there is no official competition. Not sure if "standard library" is fitting either.